data/incipias/incipias first contact.txt

```diff
@@ -264,6 +270,7 @@ mission "Incipias: Help The Stranded 1"
 				`	"I want to help one of your stranded ships."`
 			`	Again a long pause. "Your intention-goals are welcome. Please dock to the capital-station." Along the message comes a small map of the station with the docking ports highlighted.`
 	on complete
+		clear "incipias: tech dialog"
```

Comment:
a bit more descriptive name as this sounds like it is a dialog with the incipias
"incipias: quarg tech dialog"